draft-gruessing-moq-requirements.md

```diff
@@ -206,15 +207,25 @@ Our goal in this section is to understand the requirements that result from the
 
 ## Specific Protocol Considerations {#proto-cons}
 
-### Common push protocol for ingest and distribution
+In order to support the various topologies and patterns of media flows with the protocol, the protocol MUST support both sending and receiving of media streams, as separate actions or concurrently in a given connection.
 
-### Allow configurable latency - live streaming , interactive, media conferencing
+### Delivery Assurance vs. Delay
 
-### Support media transport mapping over QUIC Streams/QUIC Datagram (needs further evaluation)
+Different use cases have varying requirements with respect to the tradeoffs associated in having guarantee of delivery vs delay - in some (such as telefony) it may be acceptable to drop some or all of the media as a result of changes in network connectivity, throughput, or congestion whereas in other scenarios all media must arrive at the receiving end even if delayed. There SHOULD be support for some means for a connection to signal which media may be abandoned, and behaviours of both senders receivers defined when delay or loss occurs. Where multiple variants of media are sent, this SHOULD be done so in a way that provides pipelining so each media stream may be processed in parallel.
 
-### Support Webtransport/H3/Raw QUIC as media transport
+### Support Webtransport/Raw QUIC as media transport
 
-### Supports media over QUIC Streams/QUIC Datagrams
+Many of the use cases will be deployed in contexts where web browsers are the common application runtime; thus the use of existing protocols and APIs is desireable for implementations. Support for WebTransport {{I-D.draft-ietf-webtrans-overview}} SHOULD be defined, however SHOULD NOT be required by all implementations or deployments.
```

Comment:
> Support for WebTransport {{I-D.draft-ietf-webtrans-overview}} SHOULD be defined, however SHOULD NOT be required by all implementations or deployments.

Does this make sense to you? I can follow it, but (1) this is where BCP 14 language drives us crazy, and (2) Is your understanding that what we're doing here is defining an encapsulation that would work for either WebTransport or QUIC as the transport?

It might be useful to mention that other use cases would not be deployed within web browsers, so the other requirement is for support of QUIC transport, rather than not requiring support for WebTransport.

Reply:
> what we're doing here is defining an encapsulation that would work for either WebTransport or QUIC as the transport?

Yes, I didn't get in far enough to put that in yet, sorry.

Reply:
That's not a problem! I know I'm reviewing a moving target.

Reply:
I've added a bit of text above that talks of decoupling MoQ from the protocol, both from the use of "RAW QUIC"/H3/WebTransport, but also protocol version. I think this might be enough to give context and leave room for the discussions. Also cleaned up the normative.